lib/pub_sub/message.rb

```diff
@@ -17,12 +17,6 @@ def process
 
     def validate_message!
       messages = PubSub.config.subscriptions[sender]
-      if messages.nil?
```

Comment:
Search service was backlogged quite badly because senders mismatched. After some thought, I realized there was nothing to be gained by doing that check - other than potentially stalling the queue.